pkg/identityserver/compat/compat.go

```diff
@@ -0,0 +1,60 @@
+// Copyright © 2019 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package compat
```

Comment:
The compatibility here, for now, fully related to The Things Gateway. Should more Account Server endpoints be required for whatever reason in the future, they can be added here. I'm open for suggestions if you have something more appropriate in mind.

Reply:
@johanstokking what do you think about this? 

- Should we just put all "compatibility" stuff in the `compat` package?
- Should we put it in a `thethingsgateway` package?
- Should we update The Things Gateway firmware to support v3 endpoints?

Reply:
Discussed this offline with @johanstokking, the 3rd bullet definitely isn't a feasible option.

Considering that this functionality does almost the same as `pkg/basicstation/cups` (#75), I think it would be nice to do something similar, and to use the package `pkg/thethingsgateway/cups` for the functionality that configures and updates The Things Gateway.